tests/support/kubernetes.py

```diff
@@ -580,6 +755,177 @@ async def readline() -> bytes:
         response.content.readline.side_effect = readline
         return response
 
+    # INGRESS API
+
+    async def create_namespaced_ingress(
+        self, namespace: str, body: V1Ingress
+    ) -> None:
+        """Create an ingress object.  This will be given a dummy value
```

Comment:
Nit: Second sentence isn't part of the summary and should be a separate paragraph.